bluemira/codes/_freecadapi.py

```diff
@@ -1123,6 +1123,22 @@ def fix_wire(wire, precision=EPS, min_length=MINIMUM_LENGTH):
     wire.fix(precision, min_length, min_length)
 
 
+def fix_face(face, precision=EPS, min_length=MINIMUM_LENGTH):
```

Comment:
I don't think we need to be specific about this fixing a face - the `fix` method lives on `Part.Shape`, so any of the apiWire/apiFace/apiShell/apiSolid would work in this case. Perhaps we can just rename the one above to `fix_shape` and update the docstring to accept any shape?